soccer/gameplay/skills/benchmark/motion_benchmark.py

```diff
@@ -0,0 +1,390 @@
+            self.noiseMeasured = True
+            self.noiseStartTime = time.time()
+        deltaX = self.noiseStartPos.x - self.robot.pos.x 
+        deltaY = self.noiseStartPos.y - self.robot.pos.y
+        if(deltaX > self.noiseMaxX):
+            self.noiseMaxX = deltaX
+        if(deltaX < self.noiseMinX):
+            self.noiseMinX = deltaX
+        if(deltaY < self.noiseMinY):
+            self.noiseMinY = deltaY
+        if(deltaY > self.noiseMaxY):
+            self.noiseMaxY = deltaY
+
+    #End noise state functions
+
+    #move1 functions START
+    '''
+    #This functionality needs to be moved into the visionTest class
+    '''
+    def on_enter_move1(self):
+        self.moveStartTime = time.time()
+        self.remove_all_subbehaviors()
+        self.add_subbehavior(skills.move.Move(robocup.Point(0, constants.Field.Width / 2)), 'move')
+
+
+    def execute_move1(self):
+        noiseTestDone = self.brokenNoise()
+        if(noiseTestDone):
+            self.remove_all_subbehaviors()
+
+    def on_exit_move1(self):
+        self.moveEndTime = time.time()
+        self.remove_all_subbehaviors() 
+        self.noiseResult = abs(self.moveStartTime - self.moveEndTime)
+        print("------------------LATENCY TEST RESULTS---------------------")
+        print("Latency (seconds) = " + str(self.noiseResult))
+        print("X noise = " + str((abs(self.noiseMaxX) + abs(self.noiseMinX))))
+        print("Y noise = " + str((abs(self.noiseMaxY) + abs(self.noiseMinY))))
+        print("-----------------------------------------------------------")
+        self.currentTest.setupTest()
+
+    '''
+
+
+    def done_with_setup(self):
+        self.isSetup = True
+
+    def on_enter_TestMotion(self):
+        self.currentTest.startMotion()
+        if (self.currentTest.currentEnd is not None):
+            self.add_subbehavior(skills.move.Move(self.currentTest.currentEnd), 'move')
+        else:
+            print("There is no current end")
+
+        if (self.currentTest.currentFacePoint is not None):
+            self.robot.face(self.currentTest.currentFacePoint)
+
+    def execute_TestMotion(self):
+        if (self.currentTest.currentFacePoint is not None):
+            self.robot.face(self.currentTest.currentFacePoint)
+        self.currentTest.processMotion()
+
+    def on_enter_TestBuffer(self):
+        self.currentTest.startMotionPause()
+
+    def on_exit_TestMotion(self):
+        self.currentTest.endMotion()
+        self.remove_all_subbehaviors()
+
+    def on_enter_TestEnd(self):
+        print("Ending test " + self.currentTest.title)
+        self.currentTest.processResults()
+        pass
+
+    def on_exit_TestEnd(self):
+        self.testIndex += 1
+        if(self.testIndex < len(self.tests)):
+            self.currentTest = self.tests[self.testIndex]
+            #self.currentTest.setupTest()
+        else:
+            self.currentTest = None
+
+    #We need to
+    def on_enter_ProcessAllTests(self):
```

Comment:
I don't know what you intended to do here, but delete extra comments after you do